spec/grape/exceptions/body_parse_errors_spec.rb

```diff
@@ -52,6 +52,43 @@ def app
     end
   end
 
+  context 'api with rescue_from :grape_exceptions handler' do
+    subject { Class.new(Grape::API) }
+    before do
+      subject.rescue_from :all do |_e|
+        rack_response 'message was processed', 400
+      end
+      subject.rescue_from :grape_exceptions
+
+      subject.params do
+        requires :beer
+      end
+      subject.post '/beer' do
+        'beer received'
+      end
+    end
+
+    def app
+      subject
+    end
+
+    context 'with content_type json' do
+      it 'returns body parsing error message' do
+        post '/beer', 'test', 'CONTENT_TYPE' => 'application/json'
+        expect(last_response.status).to eq 400
+        expect(last_response.body).to include 'message body does not match declared format'
```

Comment:
The full body is 
```
Problem:
  message body does not match declared format
Resolution:
  when specifying application/json as content-type, you must pass valid application/json in the request's 'body'
```

Kind of a pain to match the full thing.   Totally could do.   Was following the precedent set on the tests below:  
https://github.com/mmclead/grape/blob/68717c623ef5c3c0ad4f9180c5553c8ce758fa92/spec/grape/exceptions/body_parse_errors_spec.rb#L111